Resources/Locales/en.json

```diff
@@ -9855,7 +9855,27 @@
           "#{Name.name}",
           "#{Company.name}"
         ]
-      }
+      },
+      "hobbit": [
+        {
+          "character":["Bilbo Baggins", "Bungo Baggins", "Belladonna Took", "Bullroarer Took", "Gandalf The Grey", "Radagast", "Dain", "Thorin Oakenshield", "Fili", "Kili", "Balin", "Dwalin", "Oin", "Gloin", "Dori", "Nori", "Ori", "Bifur", "Bofur", "Bombur", "Elrond", "Galion", "Bard the Bowman", "Beorn", "Tom", "Bert", "William (Bill Huggins)", "Gollum", "The Necromancer", "Smaug", "Carc", "Roac", "The Lord of the Eagles", "The Great Goblin", "Bolg", "Golfimbul"],
```

Comment:
@jeroendevrind Might be a bit more legible to have each array item on a new line